bin/rialto/runtime/src/lib.rs

```diff
@@ -1143,7 +1228,7 @@ mod tests {
 
 	#[test]
 	fn call_size() {
-		const MAX_CALL_SIZE: usize = 230; // value from polkadot-runtime tests
+		const MAX_CALL_SIZE: usize = 232; // slightly bigger than polkadot-runtime tests
```

Comment:
I know that BEEFY has not yet reached the Polkadot or Kusama runtimes => it is safe to ignore this limit here. Bt I suggest to add an issue to BEEFY tracker to box large call arguments and revert this change when it's done. Otherwise we'll need to have some justification of why 232 is better than 230 when BEEFY reach Polkadot/Kusama :)

Meanwhile could you, please, return `230` here and replace blanket check with checks specific to pallets that are/will be included there soon(ish):
```rust
		assert!(core::mem::size_of::<pallet_bridge_grandpa::Call<Runtime>>() <= MAX_CALL_SIZE);
		assert!(core::mem::size_of::<pallet_bridge_messages::Call<Runtime>>() <= MAX_CALL_SIZE);
```